.buildbot.sh

```diff
@@ -17,6 +17,8 @@ export RANLIB=$CHERI/bin/llvm-ranlib
 export LLVM_LINK=$CHERI/bin/llvm-link
 export PYTHONPATH=~/build/test-scripts
 export SSHPORT=10021
+export SSH_OPTIONS='-o "StrictHostKeyChecking no"'
```

Comment:
Why do we need these? Is it to avoid "The authenticity of host '127.0.0.1 (127.0.0.1)' can't be established." ? If so, is there another way to do this? e.g. does cheribuild put around a copy of a key somewhere that we can use here?

Reply:
This is the work around we use in `cheri-examples`. IIRC, I could not find another way...

Reply:
Can we ask on cheri slack to see if there's a solution? There is a definite attack (mostly an accidental attack!) where we *think* we've started something on port 10021 but actually we're just connecting to a previously running instance. If there's a nice way around this, it would be good to use it.

I won't hold up this PR now, though, so we can ask on CHERI Slack in parallel.